Makefile

```diff
@@ -1,7 +1,12 @@
-# Variables are documented in hack/build.sh.
+# Variables are documented in common/build.sh.
 BASE_IMAGE_NAME = postgresql
 VERSIONS = 9.2 9.4 9.5
 OPENSHIFT_NAMESPACES = 9.2
 
 # Include common Makefile code.
-include hack/common.mk
+-include common/common.mk
+git_submodule_warning:
```

Comment:
I've noticed that the first commit pointed to "ssh-cloned" submodule, that was wrong because not everybody (nobody except for me in case of `praiskup` namespace) is able to clone like that. Moved to https and force pushed.